common/views/components/NewsletterPromo/NewsletterPromo.tsx

```diff
@@ -38,13 +32,11 @@ const ShameButtonWrap = styled.div`
   }
 `;
 
-const NewsletterForm = styled(Space).attrs({
-  as: 'form',
+const NewsletterForm = styled.form.attrs({
```

Comment:
I changed this because the compiler was unhappy about the `noValidate` attribute being passed when we construct this component; there might be a better approach.

I tried to test I could still sign up locally, but I kept getting an error. Looking in the web console, I get a "Could not connect to the server", but the generated HTML is identical. I wonder if it's an HTTP/HTTPS thing?